Resources/Prototypes/_EE/Entities/Structures/Doors/Airlocks/shuttleAirlocks.yml

```
@@ -0,0 +1,81 @@
+# Unlimited, these are targets for shuttles to dock to.
+- type: entity
+  parent: AirlockGlassShuttle
+  id: AirlockExternalGlassShuttleCargoShuttle
+  suffix: External, Cargo, Glass, Docking
+  components:
+  - type: PriorityDock
```

YAMLlint (1.35.1)

[error] wrong indentation: expected at least 3

(indentation)